LoopKitUI/View Controllers/ServiceViewController.swift

```diff
@@ -8,24 +8,25 @@
 
 import LoopKit
 
-open class ServiceViewController: UINavigationController, ServiceSetupNotifying, ServiceSettingsNotifying, CompletionNotifying {
-
-    public weak var serviceSetupDelegate: ServiceSetupDelegate?
-
-    public weak var serviceSettingsDelegate: ServiceSettingsDelegate?
-
+open class ServiceViewController: UINavigationController, ServiceCreateNotifying, ServiceOnboardNotifying, CompletionNotifying {
+    public weak var serviceCreateDelegate: ServiceCreateDelegate?
+    public weak var serviceOnboardDelegate: ServiceOnboardDelegate?
     public weak var completionDelegate: CompletionDelegate?
 
     public func notifyServiceCreated(_ service: Service) {
-        serviceSetupDelegate?.serviceSetupNotifying(self, didCreateService: service)
+        serviceCreateDelegate?.serviceCreateNotifying(self, didCreateService: service)
     }
 
-    public func notifyServiceDeleted(_ service: Service) {
-        serviceSettingsDelegate?.serviceSettingsNotifying(self, didDeleteService: service)
+    public func notifyServiceSetup(_ service: Service) {
+        serviceOnboardDelegate?.serviceOnboardNotifying(self, didOnboardService: service)
+    }
+
+    public func notifyServiceCreatedAndSetup(_ service: ServiceUI) {
+        notifyServiceCreated(service)
+        notifyServiceSetup(service)
```

Comment:
[nit] should `setup` be replaced by `onboard` to keep the terms consistent?

Reply:
Ah, yes, good catch. Will update.